.github/workflows/verify-build.yml

```diff
@@ -30,10 +29,7 @@ jobs:
       - uses: actions/setup-go@v5
         name: Install Go
         with:
-          go-version: ${{ env.GO_VER }}
-      - name: Check Go module version
-        run: |
-          [[ "$(go list -m -f '{{.Version}}' go)" == "${GO_VER}" ]]
```

Comment:
Check that the CI and go.mod versions match is no longer required since the CI uses the go.mod version.